nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -170,6 +170,9 @@ let
             ssl_certificate ${vhost.sslCertificate};
             ssl_certificate_key ${vhost.sslCertificateKey};
           ''}
+          ${optionalString vhost.forceSSL ''
+            add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
```

Comment:
Yes, I agree that some people might want that. At the same time my point is that, if not for testing, compulsory redirect without HSTS is useless in terms of security.

How would you suggest to implement that? Have a completely separate config option? Have a separate config option, enabled by default by `forceSSL` and mentioned in its documentation? Other ideas?